cdap-api/src/main/java/co/cask/cdap/internal/workflow/ProgramWorkflowAction.java

```diff
@@ -40,9 +41,10 @@
 
   private final String name;
   private String programName;
-  private Callable<RuntimeContext> programRunner;
+  private Runnable programRunner;
   private SchedulableProgramType programType;
   private WorkflowContext context;
+  private WorkflowToken token;
```

Comment:
oops... sorry!! this was missed by me. We do not need `WorkflowToken` here and context too. Removed both.